src/stripe-config.js

```diff
@@ -0,0 +1,295 @@
+/*
+ * Stripe related configuration.
+ */
+
```

Comment:
It might be best that we add more comments to this key. We have had a couple of problems when customizers haven't really understood that Stripe has a publishable key and private key - and they both should be set. So, repeating that info probably helps.

```
// NOTE: REACT_APP_STRIPE_PUBLISHABLE_KEY is mandatory environment variable. 
// This variable is set in a hidden file: .env
// To make Stripe connection work, you also need to set Stripe's private key in the Flex Console.
```